### DISCUSSION TITLE
firmware update problem

### DISCUSSION THREAD

**Discussion by KBLiveSolutions** [KBLiveSolutions/open.control#1] [Troubleshooting] [answered]
Edited to reflect issue below

**Comment by agrandia** [comment]
Hello,
my boxed Open-control has arrived today.
I've already run several settings on ableton, it's great!
but I could not update the firmware. I followed the instructions and after copying the file inside the root of the driver it does not restart and in the editor still appears firmware version 1.2
I am a PC windows user.
what am i doing wrong?
thanks

**Reply by KBLiveSolutions** [reply]
Hi
Did you see the RPI-RP2 drive appear ?
All the LEDs of the controller should be off when you are in this mode.
I'll see if there is any difference on Windows, but the fact that it reboots when dropping a .uf2 file shouldn't be dependant on the 
OS.

Also, did you refresh the editor page ?

Meanwhile you can still use open·control, the bugfixes in 1.3 are minor.
Cheers
P-A

**Comment by agrandia** [comment]
Hello,
then I send a screenshot of what appears on my PC. The device does not restart or update
![opencontrolUF](https://user-images.githubusercontent.com/100170377/155191999-fd8662b5-efd7-419e-90d7-aee54907e692.jpg)
.

**Reply by KBLiveSolutions** [reply]
I have moved this conversation to the troubleshooting section.
I'll have a look at what happens on Windows asap. On MacOS, the .uf2 file doesn't show.

**Reply by agrandia** [reply]
OK,
There is also a problem displaying the Control Surface in Live. I leave a picture
![Sense títol](https://user-images.githubusercontent.com/100170377/155217927-07274eca-0b6a-49f1-af9b-7d4e701aeb74.jpg)

**Reply by KBLiveSolutions** [reply]
Weird. It was working fine yesterday if I check your first post ?
Did you make any changes ?

**Reply by agrandia** [reply]
Well, for now, I've only used it in Custom Midi settings. I haven't experimented with Ableton's native features.
I can send information and get Ableton to receive both CC and Note.
There is also some functionality in the WEB editor that does not just refresh properly.
I will report soon.
I also wanted to ask if in any front slot (1-6) we can put an expression pedal and set it to send CC.
Have you tried different types of sustain pedals? Which are the best?

**Reply by KBLiveSolutions** [reply]
Ok, thanks for the feedback, I understand better. 
Please be patient, although everything was carefully beta-tested, the project is still very fresh. I will make sure every issue is addressed.

Can you please send me the Log.txt file. You will find it here: 
\User\[user name]\AppData\Roaming\Ableton\Live x.x.x\Preferences\Log.txt
Please send it at : contact (at) kblivesolutions.com

It is not possible to use the front slots to use Expression Pedals, only switches are accepted.
I haven't tried many switches. The cheapest ones on Thomann are pretty fine imo.

**Reply by KBLiveSolutions** [reply]
Thanks for the Log.txt.
 I think I know what's wrong.
The folder you have to copy inside the Remote Script folder is the folder named "open_control", which is INSIDE the unzipped folder "open.control-remote_script-main"

**Comment by catnapster** [comment]
This is all very exciting. I've ordered mine already - just the basic to start, but I will probably also buy a dev kit in the future, if this works out. 

Just wanted to say hello and suggest that the firmware upgrade be turned into a different post, so that people in the future who are looking for questions about firmware upgrades have a good thread to search for. 

I am super stoked about this whole project :)

**Reply by KBLiveSolutions** [reply]
Thanks for your message!
I moved the firmware issue to the troubleshooting section, and thus moved your message. If you want you can re-post it in the new [Welcome message](https://github.com/KBLiveSolutions/open.control/discussions/10) I have created.
This is already becoming messy ! :-)

**Reply by KBLiveSolutions** [reply]
Can I ask you if you ordered it on Kickstarter or on the website ?
Have you already received it ?

**Reply by KBLiveSolutions** [reply]
Hi Catnapster
I am getting back to you because I'm not sure about your message : are you still waiting for a unit ? 
I don't have any pending order, so you should already have received it. Is it the case ?
Please get in touch with me : contact (at) kblivesolutions.com

**Comment by KBLiveSolutions** [comment, marked as answer]
This was handled one-to-one, case solved. :-)

**Reply by agrandia** [reply]
I have to say, there is a wonderful person behind the Open.control project. During the process of learning how to update my device I found the support, interest and constant smile of a person who has been 100% helpful. Thank you so much!!!
